src/selectors/per-thread/markers.js

```diff
@@ -216,37 +264,56 @@ export function getMarkerSelectorsPerThread(threadSelectors: *) {
     }
   );
 
+  /**
+   * This selector finds out whether there's any network marker in this thread.
+   */
   const getIsNetworkChartEmptyInFullRange: Selector<boolean> = createSelector(
     getFullMarkerList,
     markers => markers.every(marker => !MarkerData.isNetworkMarker(marker))
   );
 
-  const getNetworkChartMarkerIndexes: Selector<MarkerIndex[]> = createSelector(
+  /**
+   * This selector filters network markers from the range filtered markers.
+   */
+  const getNetworkMarkerIndexes: Selector<MarkerIndex[]> = createSelector(
```

Comment:
Writing comments, I realized that the same selector was present twice. So I deduplicated here.